src/async_fuse/memfs/kv_engine/etcd_impl.rs

```diff
@@ -207,11 +531,37 @@ impl KVEngine for EtcdKVEngine {
         }
     }
 
+    /// Range get, return all key-value pairs start with prefix
     async fn range(&self, prefix: &KeyType) -> DatenLordResult<Vec<ValueType>> {
         let _timer = KV_METRICS.start_kv_operation_timer("range");
         let result = self.range_raw_key(prefix.to_string_key()).await?;
         Ok(result)
     }
+
+    /// Create a lease
+    async fn create_lease(&self, ttl: i64) -> DatenLordResult<i64> {
+        let _timer = KV_METRICS.start_kv_operation_timer("lease");
+        let ttl = check_ttl(ttl).with_context(|| "failed to check ttl".to_owned())?;
+        let mut client = self.client.clone();
+        let resp = client
+            .lease_grant(ttl, None)
+            .await
+            .with_context(|| "failed to get lease at `MetaTxn::create_lease`".to_owned())?;
+        Ok(resp.id())
```

Comment:
Should be `create_session`. We should not expose lease to user,  return a Session.

Reply:
Ok, I have remove this function and use create_session instead.